pkg/grpc/auditlogstore/audit_log_store.go

```diff
@@ -54,7 +53,7 @@ func (s store) List(_ grpccontext.Context, _ *audit_log_protos.ListRequest) (*au
 	}, nil
 }
 
-func (s store) Delete(ctx grpccontext.Context, req *audit_log_protos.DeleteRequest) (*audit_log_protos.DeleteResponse, error) {
```

Comment:
unfortunately I dont think this will compile, this `store` type needs to implement `audit_log_protos.P2AuditLogStoreServer` and in order to implement an interface you have to take EXACTLY the same types as arguments even if the types are the same shape. In other words, `"google.golang.org/x/net/context" != "context"`